### PR TITLE
Combine buildTarget and additionalBuiltTargets into buildTargets

### PR DESCRIPTION
**Pull request by bennettp123**
Deprecates buildTarget and additionalBuiltTargets, and combines them into buildTargets

### CODE REVIEW

File: libs/pulumi/src/executors/up/executor.ts [hunk 1]
```diff
@@ -26,39 +26,36 @@ export default async function runUpExecutor(
...
+        project: options.targetProjectName,
+        target: options.buildTarget ?? 'build',
+        configuration: 'production',
+    }] : []
+    const deprecatedAdditionalBuildTargets
+        = options.additionalBuildTargets ?? []
+
+    for (const buildTarget of [
+        ...buildTargets,
+        ...deprecatedBuildTarget,
```
Comment (JakeGinnivan) [inline libs/pulumi/src/executors/up/executor.ts:41]: TBH i'd just make it a breaking change, the lib isn't at v1 so we can make breaking changes

File: .changeset/eighty-roses-jog.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@wanews/nx-pulumi': major
+---
+
+Deprecate buildTarget and additionalBuiltTargets, and combines them into buildTargets
```
Comment (JakeGinnivan) [inline .changeset/eighty-roses-jog.md:5]: Say `BREAKING:`

Then add instructions to migrate. 

In fact you can automate the migration if you want with NX

File: libs/pulumi/src/executors/up/executor.ts (hunk 1 above)
Comment (bennettp123) [reply libs/pulumi/src/executors/up/executor.ts:41]: I know, I just don't really want to redo what's already in workspace.json in other projects 😛

File: libs/pulumi/src/executors/up/schema.d.ts [hunk 3]
```diff
@@ -1,11 +1,22 @@
...
     targetProjectName: string
     configurationStackFormat?: string
-    buildTarget?: string
-    additionalBuildTargets?: Array<{
-        project: string
-        target: string
-        configuration?: string
-    }>
     infrastructureProject?: string
+    buildTargets?: Array<BuildTarget>
```
Comment (JakeGinnivan) [inline libs/pulumi/src/executors/up/schema.d.ts:11]: Need to also update the JSON Schema file

File: libs/pulumi/src/executors/up/executor.ts (hunk 1 above)
Comment (bennettp123) [reply libs/pulumi/src/executors/up/executor.ts:41]: An nx migration would help

File: libs/pulumi/src/executors/up/schema.d.ts (hunk 3 above)
Comment (bennettp123) [reply libs/pulumi/src/executors/up/schema.d.ts:11]: :drake-like:

**Pull request closed without merging**